APIs/apitransformer.com/patch.yaml

```diff
@@ -2,6 +2,7 @@ info:
   x-unofficialSpec: true
   x-apisguru-categories:
     - developer_tools
+    - tools
```

Comment:
Don't think it worth to use both categories at the same time
developer_tools should be enough